utam-core/src/main/java/utam/core/driver/Document.java

```diff
@@ -36,4 +36,12 @@
    * @return true if element found
    */
   boolean containsElement(Locator locator);
+  
+  /**
+   * Trigger the onChange event for the element with the given locator 
+   * in the dom, since appium doesn't use the native keyboard
+   * 
+   * @param locator locator to find an element
+   */
+  void triggerChangeEvent(Locator locator);
```

Comment:
This looks like we're adding a new grammar element. Have we had a full architectural discussion, with an RFC approved for it? I don't remember us having one. Also, this solution seems to be counterintuitive. A user would have to add a call to the action in a method in the PO JSON, but consider this JSON:

```json
{
  "methods": [
    {
      "name": "setMyFieldValue",
      "compose": [
        {
          "element": "textField",
          "apply": "setText",
          "args": [
            {
              "name": "fieldValue",
              "type": "string"
            }
          ]
        },
        {
          "element": "document",
          "apply": "triggerChangeEvent",
          "args": [
            {
              "value": { "css": "input[type='text']" }
            }
          ]
        }
      ]
    }
  ],
  "elements": [
    {
      "name": "myField",
      "public": true,
      "type": "editable",
      "selector": {
        "css": "input[type='text']"
      }
    }
  ]
}
```

This would generate, in the java class, the following interface:

```java
public interface MyInterface extends PageObject {
    void setMyFieldValue(String fieldValue);
    Editable getMyField();
}
```

This is incredibly confusing to a user. How are they supposed to know whether to use `getMyField().setValue()` or `setMyFieldValue()`? Plus, using the user-defined method should _only_ be required in the iOS case, right?

I can't help but think there must be a better way to have the mobile element fire the `onchange` event when it's in the proper context without the user needing to know to do so. It seems to me like code to detect that the element that `setValue` is being called on should be able to detect that the event should be fired (that it's in the iOS-specific environment where the driver bug exists), and just fire the event there.